content/en/security_platform/cspm/getting_started.md

```diff
@@ -19,30 +19,200 @@ Cloud Security Posture Management is not currently available in this site.
+
+### Security Setup & Configuration
+
+1. Navigate to **Security** > **Setup and Configuration**.
+2. Follow the [in-app instructions][3] to activate CSPM for your account.
+3. On the **Setup and Configuration** > **Cloud Providers** tab, click the **GCP** tile.
+4. Enable CSPM for your GCP projects by turning on the **CSPM Enabled** toggle.
+
+### GCP integration tile
+
+1. On the GCP integration tile, select a GCP project.
+2. Under **Enable resource collection for Cloud Security Posture Management**, select the **Resource collection** checkbox.
+3. Click **Update Configuration**.
+
+**Note**: To disable CSPM, turn off the **CSPM Enabled** toggle for your GCP projects. Your previous findings and homepage will still be available in-app, but you will not incur additional billing costs.
+
+[1]: https://docs.datadoghq.com/integrations/google_cloud_platform
+[2]: https://docs.datadoghq.com/integrations/google_cloud_platform/#metric-collection
+[3]: https://app.datadoghq.com/security/configuration
+{{% /tab %}}
+
+{{% tab "Docker" %}}
+
+### Enable CSPM for Docker
+
+1. Navigate to **Security** > **Setup and Configuration**.
+2. Follow the [in-app instructions][1] to activate CSPM for your account.
+3. On the **Setup and Configuration** > **Host and containers** tab, click the **Docker** tile.
+4. Click **Select API key** to choose the API key you want to use with CSPM.
+5. Run the following command to enable CSPM in your Docker environment:
+
+    ```bash
+    DOCKER_CONTENT_TRUST=1 \
+      docker run -d --name dd-agent \
+      --security-opt apparmor:unconfined \
+      --cap-add SYS_ADMIN \
+      --cap-add SYS_RESOURCE \
+      --cap-add SYS_PTRACE \
+      --cap-add NET_ADMIN \
+      --cap-add IPC_LOCK \
+      -v /var/run/docker.sock:/var/run/docker.sock:ro \
+      -v /proc/:/host/proc/:ro \
+      -v /sys/fs/cgroup/:/host/sys/fs/cgroup:ro \
+      -v /etc/passwd:/etc/passwd:ro \
+      -v /etc/group:/etc/group:ro \
+      -v /:/host/root:ro \
+      -v /etc/os-release:/host/etc/os-release \
+      -v /sys/kernel/debug:/sys/kernel/debug \
+      -e DD_COMPLIANCE_CONFIG_ENABLED=true \
+      -e DD_RUNTIME_SECURITY_CONFIG_ENABLED=true \
+      -e DD_SYSTEM_PROBE_ENABLED=true \
+      -e HOST_ROOT=/host/root \
+      -e DD_API_KEY=<DATADOG_API_KEY> \
+      -e DD_SITE="datadoghq.com" \
+      datadog/agent:latest
+    ```
+
+**Note**: To disable CSPM, set `DD_COMPLIANCE_CONFIG_ENABLED` to `false`. Your previous findings and homepage will still be available in-app, but you will not incur additional billing costs.
+
+[1]: https://app.datadoghq.com/security/configuration
+
+{{% /tab %}}
+
+{{% tab "Kubernetes" %}}
+
+### Enable CSPM for Kubernetes
+
+1. If you haven't already, install the [Datadog Agent][1] (version 7.27+).
+2. Navigate to **Security** > **Setup and Configuration**.
+3. Follow the [in-app instructions][2] to activate CSPM for your account.
+4. Add the following to the `datadog` section of the `values.yaml` file:
```

Comment:
Noting that the code sample includes a code comment about enabling CWS. Do we want to keep that in this article?